### ISSUE TITLE
Verifying Deployment not working

### ISSUE THREAD

**Issue by Slind14** [zodern/meteor-up#1110]
Hi,
we added a network to the configuration which resulted in this bug: https://github.com/zodern/meteor-up/issues/1096

After updating to 1.5-beta5 this was resolved. Unfortunately, we are now getting this (previously without the network config it was working fine):

```
[5.9.69.72] - Start Meteor: SUCCESS
[5.9.69.72] - Verifying Deployment
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:25 --:--:--     0
curl: (28) Connection timed out after 25001 milliseconds
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:25 --:--:--     0
curl: (28) Connection timed out after 25001 milliseconds
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:25 --:--:--     0
curl: (28) Connection timed out after 25001 milliseconds
=> Container status:
restarted: 0 times {"Bridge":"","SandboxID":"9e43ca54cdabe3336fa47b2ecb98c432c59dfe9727efc14da5f9a79ad325f5b9","HairpinMode":false,"LinkLocalIPv6Address":"","LinkLocalIPv6PrefixLen":0,"Ports":{"3000/tcp":[{"HostIp":"0.0.0.0","HostPort":"5000"}],"80/tcp":null},"SandboxKey":"/var/run/docker/netns/9e43ca54cdab","SecondaryIPAddresses":null,"SecondaryIPv6Addresses":null,"EndpointID":"9fdeefd12e8bd0fe4a8fef33f695c583230715323b71f9e8c41d8f884dab2ad6","Gateway":"172.17.0.1","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"IPAddress":"172.17.0.3","IPPrefixLen":16,"IPv6Gateway":"","MacAddress":"02:42::11:00:03","Networks":{"ay_ch":{"IPAMConfig":{"IPv4Address":"10.0.0.97"},"Links":null,"Aliases":["82b5a434c49d"],"NetworkID":"pdapc475iw5ejq00xudbolzil","EndpointID":"3417882cb029ff9b116f14c88da3bd1eecb2b00ee5f673fd6e38f2db5f425b55","Gateway":"","IPAddress":"10.0.0.97","IPPrefixLen":24,"IPv6Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"MacAddress":"02:42::00:00:61","DriverOpts":{}},"bridge":{"IPAMConfig":null,"Links":null,"Aliases":null,"NetworkID":"f92f00c4a5d8aae5bab12e04656606503595feb9b1c07e0686911d691e6528c3","EndpointID":"9fdeefd12e8bd0fe4a8fef33f695c583230715323b71f9e8c41d8f884dab2ad6","Gateway":"172.17.0.1","IPAddress":"172.17.0.3","IPPrefixLen":16,"IPv6Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"MacAddress":"02:42::11:00:03","DriverOpts":null}}} {"Status":"running","Running":true,"Paused":false,"Restarting":false,"OOMKilled":false,"Dead":false,"Pid":29635,"ExitCode":0,"Error":"","StartedAt":"2019-10-23T14:33:48.422480671Z","FinishedAt":"0001-01-01T00:00:00Z"}
=> Logs:
=> Starting meteor app on port:3000
Monti APM: completed instrumenting the app
Monti APM: Successfully connected

=> Redeploying previous version of the app


To see more logs type 'mup logs --tail=200'
```

**Comment by Slind14** [comment]
Changing it to `CONTAINER_IP=$(sudo docker inspect --format='{{.NetworkSettings.IPAddress}}' $APPNAME | head -n 1)` resolved it for us.

**Comment by zodern** [comment]
Could you please try running this command on the server, and checking how the output differs from what you changed it to:
```
sudo docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}
  {{end}}' <app name> | head -n 1
```

**Comment by Slind14** [comment]
`sudo docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}} {{end}}' <app name> | head -n 1`

> 10.0.0.2

`sudo docker inspect --format='{{.NetworkSettings.IPAddress}}' <app name> | head -n 1`

> 172.17.0.4

**Comment by zodern** [comment]
That is interesting. Sometimes `NetworkSettings.IPAddress` is empty though (for example, when the container is not connected to the bridge network), so your PR should check for that and fall back to how it currently works on the dev branch.

**Comment by zodern** [comment]
This will be fixed in the next 1.5 beta

**Comment by Slind14** [comment]
Thank you.
